packages/rancher-monitoring/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: beaa76a4ac26584c6cc77c27bc0d40769fb98b26
-version: 103.0.4
+version: 103.0.5-rc1
```

Comment:
This should be `103.1.0-rc1` now

Reply:
Working to rebase and resolve this now.

Reply:
Updated that - think it should be good now after rebase.